doc/memos/rdm1.md

```diff
@@ -0,0 +1,112 @@
+RIOT Maintainers Community (RIOT-MC)
+Developer Memo: 1
+Author(s): Hauke Petersen, more here
+Date: December 2016
+
+***
+RIOT Developer Memo Format, Publishing and Maintenance Process
+***
+
+# Abstract
+
+This memo contains a glossary with expressions commonly used in RIOT and in the
+context of RIOT. The aim of this document is not only to put some names to often
+used abbreviations, but also to explain their interpretation withing the RIOT
+community.
+
+
+# Status
+
+This document is a product of the community of RIOT maintainers, and aims to
+represent the consensus within this community.
+The content of this document is licensed with a Creative Commons CC-BY-SA
+license.
+
+# Terminology
+
+See below.
+
+
+# 1. Introduction
+
+As with many open source communities, there are a number of expressions, that
+are being commonly used among community members and community documents. Though
+most of these expressions are well known among developers, some of them are
+specific or are used with not-so-common semantic in the context of RIOT.
+
+This document provides a list of the most commonly used abbreviations and
+bash-words in RIOT. The scope of this document is not only to map these
+expressions to some more meaningful names, but also to explain their
+interpretation inside the RIOT community and to give some links to further
+information.
+
+Another aim of this RDM is further to act as a referencing target for future any
+RMDs.
+
+
+# 2. RIOT Glossary
+
+**ACK:** Acknowledgment of a pull request (PR) from a RIOT maintainer.
+
+**IoT:** Internet of Things - the thing everyone talks about theses days. In the
+context of RIOT we consider the IoT to be about a huge number of constrained,
+interconnected devices, many operated on small power budgets (e.g. battery
+driven).
+
+**PR:** Git merge request on github. If you want to get code into RIOT's master
+branch, opening a PR is the only way.
+
+**RDM:** RIOT Developer Memo - these are documents like this one, containing
+typically more high-level documentation in addition to doxygen and the wiki. See
```

Comment:
Not sure, if "high-level" is mandatory for these documents. Maybe "documentation of interfaces, concepts, and processes".

I would also replace "doxygen" by "API documentation" (doxygen is just the tool/format). You could also link to the API documentation and the wiki.